src/backend/renderer/element/surface.rs

```diff
@@ -489,6 +489,12 @@ impl<R: Renderer + ImportAll> Element for WaylandSurfaceRenderElement<R> {
     fn kind(&self) -> Kind {
         self.kind
     }
+
+    fn presentation_mode(&self) -> Option<PresentationMode> {
+        // On Wayland assume VSync as default preference
+        Some(PresentationMode::VSync)
+        // TODO: Return Async in case wp_tearing is attatched to the surface
```

Comment:
This is waiting on the tearing control PR?

Reply:
Yep, or the other way around, I don't care. Although the protocol PR is probably more merge ready. We could also just merge the PRs into one.